examples/00-mapdl-examples/BodyFlux-Averaging.py

```diff
@@ -0,0 +1,324 @@
+"""
+.. _ref_solenoid_magnetostatic_2d:
+
+=======================================
+Analysis of a 2D Magnetostatic Solenoid
+=======================================
+
+This example shows how to gather and plot results with material discontinuities
+across elements (Power graphics style) versus the default full average results
+(Power graphics style).
```

Comment:
line 10 should be (Full graphics style).

Reply:
Done.